src/basic/vx_util/ctrack.hpp

```diff
@@ -762,7 +762,9 @@ namespace ctrack {
                                          // Write out the detail information to the detail_output.txt file
                                          out << "filename: " << entry->filename 
                                                 << "|function: " << entry->function_name 
-                                                << "|line: " << entry->line 
+                                                << "|line: " << entry->line
+                                                 << "|start_time: " << BeautifulTable::table_timepoint(track_start_time)
```

Comment:
The whitespace formatting is inconsistent in this file with a mix of tabs and spaces, including this line being offset. The MET code generally uses spaces throughout. If possible, I'd recommend cleaning this up and keeping the formatting conistent.